packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -281,6 +294,82 @@ export interface AutoScalingGroupProps extends CommonAutoScalingGroupProps {
   readonly role?: iam.IRole;
 }
 
+/**
+ * Group metrics that an Auto Scaling group sends to Amazon CloudWatch.
+ */
+export class GroupMetric {
+
+  /**
+   * All available GroupMetrics.
+   */
+  public static readonly ALL = new Array<GroupMetric>();
+
```

Comment:
@iliapolo changing the array type from:

```ts
 public static readonly ALL = new Array();
```
To:

```ts
public static readonly ALL = new Array<GroupMetric>();
```
Did the trick, now `groupMetricsCollections ` can not be initialized by an empty array.